packages/web/src/pages/hustlers/[id]/flex.tsx

```diff
@@ -0,0 +1,321 @@
+        ...hustlerConfig,
+        name: h?.name || '',
+        title: h?.title || '',
+        bgColor: `#${h?.background}`,
+        sex: (h?.sex.toLowerCase() || 'male') as HustlerSex,
+        body: h.body?.id ? parseInt(h.body.id.split('-')[2]) : 0,
+        hair: h.hair?.id ? parseInt(h.hair.id.split('-')[2]) : 0,
+        facialHair: h.beard?.id ? parseInt(h.beard.id.split('-')[2]) : 0,
+      });
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [data, hustlerId]);
+
+  return (
+    <AppWindow padBody={true} scrollable title='SWAPMEET.EXE' subTitle='500k $PAPER | clicksave.eth'>
+      <Head
+        title={`SWAPMEET.EXE`}
+        ogImage={`https://api.dopewars.gg/hustlers/${hustlerId}/sprites/composite.png`}
+      />
+      {isLoading && <LoadingBlock />}
+      {!isLoading && itemRles && (
+        <Box display="flex" background={"white"} minHeight="100%" p={'5'} flexDirection={isMobile ? 'column' : 'row'}>
+          <Box flex={1} mb={isMobile ? "5" : "0"} width="full" display="flex" flexDirection={"column"}>
+            <Box display="flex" alignItems="center" justifyContent="space-between" width={'full'} mb="5">
+              <Box display="flex" alignItems="center">
+                <Box><Image src="/images/hustler/backIco.svg" alt="back" /></Box>
+                <Box marginLeft="0.5">Back</Box>
+              </Box>
+              <Box>OG Calico</Box>
+            </Box>
+            <Box display="flex" flexDirection="column" borderRadius={'3'} overflow="hidden" border="1px solid black" mb="3">
+              <Image src={onChainImage} alt="png" />
+              <Box color="#979999" width={'full'} textAlign="center" py="1" bgColor="#434345"  borderTop="1px solid black">Not For Sale</Box>
+            </Box>
+            <Box display={'flex'} justifyContent="space-between" width={'full'} mb="5">
+              <Button ><Image src="/images/hustler/download.svg" alt="download" /></Button>
+              <Button ><Image src="/images/hustler/camera.svg" alt="download" /></Button>
+              <Button onClick={()=>{router.push({pathname: `/hustlers/${router.query.id}/flex`})}}><Image src="/images/hustler/share.svg" alt="download" /></Button>
+              <Button onClick={()=>{router.push({pathname: `/hustlers/${router.query.id}/customize`})}}><Image src="/images/hustler/edit.svg" alt="download" /></Button>
+              <Button onClick={()=>{router.push({pathname: `/hustlers/${router.query.id}/gear`})}}><Image src="/images/hustler/setting.svg" alt="download" /></Button>
+            </Box>
+            <Box display={"flex"} width={'full'} mb="5">
+              <Button width={"100%"}>Create Swap</Button>
+            </Box>
+            <Box display={"flex"} width={'full'}>
+              <Button width={"100%"}>Transfer</Button>
+            </Box>
+          </Box>
+          <Box flex={isMobile ? 1 : 2} width="full" ml={isMobile ? "0" : "5"} display={'flex'} flexDirection="column">
```

Comment:
Chakra which is the UI library we're using has support for passing multiple parameters like so: https://chakra-ui.com/docs/styled-system/features/responsive-styles#the-array-syntax

Please use that array syntax instead. This "isMobile" hook is unnecessary.